app/src/main/cpp/skyline/services/bcat/IServiceCreator.h

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <services/serviceman.h>
+#include "IBcatService.h"
+
+namespace skyline::service::bcat {
+    /**
+     * @brief IApplicationProxyService is used to open an application proxy
+     * @url https://switchbrew.org/wiki/Applet_Manager_services#appletOE
```

Comment:
I assume you copy-pasted the comment here from AM, you should correct the URL and make the brief something along the lines of `IServiceCreator is used to create per-process/per-title service instances for BCAT`.